content/articles/integrated-domain-providers.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrated Domain Providers at DNSimple
+excerpt: Link an integrated domain provider to your DNSimple account to manage domains at other domain providers, from within DNSimple.
+categories:
+- Domains
+- Integrations
+---
+
+# Integrated Domain Providers
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple supports the ability to manage domains that are registered with other domain providers external to DNSimple. DNSimple calls such providers "integrated domain providers".
+
+## Supported Features
+
+- **View domain information**: When you link an integrated domain provider to your DNSimple account, domains registered via that integrated domain provider will be imported into DNSimple and listed on the Domains page.
+- **Domain expiration notices**: Get email notifications when your integrated domains are near expiry.
+
+## Supported Integrated Domain Providers
+
+To manage integrated domains from DNSimple, you first have to link an integrated domain provider to your DNSimple account.
+
+DNSimple supports domain integration with the providers listed below.
+
+- [GoDaddy](https://www.godaddy.com)
```

Comment:
I feel we should link/have another page for GoDaddy, similar to https://deploy-preview-1065--dnsimple-support.netlify.app/articles/integrated-dns-provider-amazon-route53/.

Reply:
Great idea, updated to link to a separate page for GoDaddy: https://deploy-preview-1065--dnsimple-support.netlify.app/articles/integrated-domain-provider-godaddy/